plugins/orchestrator-common/scripts/openapi.sh

```diff
@@ -2,8 +2,12 @@
 pwd
 set -ex
 
-# npx openapi typegen ./api/openapi.yaml > src/openapi/openapi.d.ts
-npx openapi-typescript ./api/openapi.yaml -o ./src/auto-generated/api/models/schema.ts
+export SCHEMA=./src/auto-generated/api/models/schema.ts
+npx openapi-typescript ./api/openapi.yaml -o ${SCHEMA} --alphabetize --enum
+# Fix sonarcloud warning typescript:S101 (Class names should comply with a naming convention)
+sed -i 's/export interface paths {/export interface paths { \/\/ NOSONAR/' ${SCHEMA}
+sed -i 's/export interface components {/export interface components { \/\/ NOSONAR/' ${SCHEMA}
+sed -i 's/export interface operations {/export interface operations { \/\/ NOSONAR/' ${SCHEMA}
```

Comment:
@caponetto do you know how to ignore SonarCloud warning for `components`,`paths` and `operations`?
 This is an auto generated code. This fix doesn't work.

Reply:
@gciavarrini IMO we don't need to work around that, i.e., we can keep the generated code as is.
I've resolved the issues with "Won't fix - Auto-generated code from OpenAPI"